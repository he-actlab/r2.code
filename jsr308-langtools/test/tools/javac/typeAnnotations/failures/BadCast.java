/*
 * @test /nodynamiccopyright/
 * @bug 1234567
 * @summary A cast cannot consist of only an annotation.
 * @author Werner Dietl
 * @compile/fail/ref=BadCast.out -XDrawDiagnostics -source 1.8 BadCast.java
 */
class BadCast {
  static void main() {
    Object o = (@A) "";
  }
}

@interface A { }
