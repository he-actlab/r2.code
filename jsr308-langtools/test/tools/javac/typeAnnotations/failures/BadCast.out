BadCast.java:10:19: compiler.err.illegal.start.of.expr
1 error